pkg/signal/trap.go

```diff
@@ -58,6 +59,9 @@ func Trap(cleanup func(), logger interface {
 						logger.Info("Forcing docker daemon shutdown without cleanup; 3 interrupts received")
 					}
 				case syscall.SIGQUIT:
+					logger.Info("Kill USR1 docker-containerd")
+					exec.Command("pkill", "-USR1", "docker-c").Run()
+					<-time.After(500 * time.Millisecond)
```

Comment:
Leftover debug code?

Reply:
Not really, in general we want both together

Reply:
This seems like the wrong place to implement this logic. `pkg/signal` shouldn't really know about the binary names.

I think `Trap()` needs to be refactored to accept a function to call when it receives `SIGQUIT`.

Reply:
Oh, completely forgot it was inside `pkg/trap`. Will remove it for now.